keyboards/planck/keymaps/snowkuma/config.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```

Comment:
`keyboards/planck/config.h` includes `config_common.h` already, so this line is redundant.

```suggestion
```